### DISCUSSION TITLE
Any idea why it wont run?

### DISCUSSION THREAD

**Discussion by Jimbok21** [cordejs/corde#1346] [Q&A] [answered]
When I try to run corde, I get this error. Not sure why this happens as all of these files exists. Below is the terminal error that I get
![image](https://user-images.githubusercontent.com/38737780/165986201-c3f2360c-c61e-419c-9fa8-0e4056fe1506.png)
Here is a screenshot to BaseClient.js that the error mentions. You can see that the file exists in the correct file path
![image](https://user-images.githubusercontent.com/38737780/165986395-ab9c07ff-3e38-4d5c-a9cb-6a68bd41a8c0.png)

I just updated node to v18. Running the bot normally works fine but using corde crashes
Any help is appreciated

Edit:
I found that this line is making it crash. I've tried changing the file path and moving the file out of the test folder but none of that works `const client = require('..')`

My app.js can be seen in my github for my Discord Bot

**Comment by lucasgmagalhaes** [comment, marked as answer]
Hi, @Jimbok21. The error was caused by `esm` module. Which is used by corde. This error started to appear in version 13 of Discord.js, so, in order to use corde, unfortunately, you would have to use v12 of discord.js

**Reply by Jimbok21** [reply]
Thanks for letting me know. I wrote about this library for my dissertation project and this is great info to include.
